packages/hardhat/contracts/CallForFundsLogic.sol

```diff
@@ -2,30 +2,26 @@
 pragma solidity ^0.8.2;
 
 import {CallForFundsStorage} from "./CallForFundsStorage.sol";
+import {CryptoCredential} from "./CryptoCredential.sol";
 
-contract CallForFundsLogic is CallForFundsStorage {
+contract CallForFundsLogic is CryptoCredential, CallForFundsStorage {
```

Comment:
I'll be honest that I don't fully get the benefit of doing this yet, I would love you to walk me through it so I can understand a bit better.